openedx/features/edly/tests/test_middleware.py

```diff
@@ -54,17 +55,25 @@ def test_user_with_edly_organization_access(self):
 
     def test_user_without_edly_organization_access(self):
         """
-        Verify that logged in user gets valid error and log message response if user has no access.
+        Verify that logged in user gets redirected to logout page and valid log message response if user has no access.
 
-        Test that logged in user gets 404 and valid log message if user has no access for
+        Test that logged in user gets redirected to logout page and valid log message if user has no access for
         request site's edly sub organization.
         """
 
         with LogCapture(LOGGER_NAME) as logger:
             response = self.client.get('/', follow=True)
-            assert response.status_code == 404
+            self.assertRedirects(
+                response, '/logout',
```

Comment:
nit: either keep all argument in one line or only one argument in one line.  https://docs.djangoproject.com/en/3.0/topics/testing/tools/#django.test.SimpleTestCase.assertRedirects
```python
response, 
'/logout',
```

Reply:
Done